parachain/pallets/outbound-queue/src/lib.rs

```diff
@@ -354,18 +358,30 @@ pub mod pallet {
 				payload.len() < T::MaxMessagePayloadSize::get() as usize,
 				SubmitError::MessageTooLarge
 			);
+
+			let base_fee = T::WeightToFee::weight_to_fee(
+				&T::WeightInfo::do_process_message()
+					.saturating_add(T::WeightInfo::commit_one_message()),
+			);
 			let delivery_fee = Self::delivery_fee(&message.command);
+
 			let command = message.command.clone();
+			let fee = Fees { base: base_fee, delivery: delivery_fee };
+
 			let enqueued_message: EnqueuedMessage =
 				EnqueuedMessage { id: message_id, origin: message.origin, command };
 			// The whole message should not be too large
 			let encoded =
 				enqueued_message.encode().try_into().map_err(|_| SubmitError::MessageTooLarge)?;
 
-			let ticket =
-				OutboundQueueTicket { id: message_id, origin: message.origin, message: encoded };
+			let ticket = OutboundQueueTicket {
+				id: message_id,
+				origin: message.origin,
+				message: encoded,
+				fee,
```

Comment:
I'm not seeing the `OutboundQueueTicket.fee` field being used anywhere. Can it removed?

Reply:
[Remove fee from OutboundQueueTicket](https://github.com/Snowfork/snowbridge/pull/975/commits/8fa7dacdbe5ea9c260b3da7af7706a8955139dec)